src/compas/robots/model/geometry.py

```diff
@@ -501,6 +535,10 @@ def __init__(self, filename):
         super(Texture, self).__init__()
         self.filename = filename
 
+    def get_urdf_element(self):
```

Comment:
shouldn't all these be named `to_urdf` or `to_urdf_element`?

Reply:
I wouldn't want a `to_urdf_element` or `to_urdf` method on `RobotModel` to be confused by a user with something that produces a urdf file.  So I prefer to leave this largely internal method name as is.